encodings/datetime-parts/src/array.rs

```diff
@@ -11,7 +11,7 @@ use vortex_error::{vortex_bail, VortexResult};
 
 use crate::compute::decode_to_temporal;
 
-impl_encoding!("vortex.datetimeparts", 20u16, DateTimeParts);
+impl_encoding!("vortex.datetimeparts", 22u16, DateTimeParts);
```

Comment:
oh whoops, this actually collides with runendbools

Reply:
lol

Reply:
https://github.com/spiraldb/vortex/pull/498